src/zrc/api/filters.py

```diff
@@ -16,6 +18,11 @@
 )
 
 
+def get_most_recent_status(queryset, name, value):
+    qs = queryset.order_by("-datum_status_gezet")[:1]
```

Comment:
return gewoon direct het resultaat van de queryset of bescherm de functie met een index check. Nu is de variable een beetje nutteloos.